### DISCUSSION TITLE
%E7%88%AC%E8%99%AB/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/%E5%B8%B8%E7%94%A8hook%E4%BB%A3%E7%A0%81%E7%89%87%E6%AE%B5

### DISCUSSION THREAD

**Discussion by giscus** [cpython666/cpython666.github.io#2] [Announcements]
# %E7%88%AC%E8%99%AB/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/%E5%B8%B8%E7%94%A8hook%E4%BB%A3%E7%A0%81%E7%89%87%E6%AE%B5

Python斗罗的代码笔记

https://cpython666.github.io/%E7%88%AC%E8%99%AB/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/%E5%B8%B8%E7%94%A8hook%E4%BB%A3%E7%A0%81%E7%89%87%E6%AE%B5.html

**Comment by cpython666** [comment]
**屌炸了**

**Reply by FCL-Python** [reply]
雀食
